src/browser/modules/Editor/EditorFrame.tsx

```diff
@@ -105,11 +111,15 @@ export function EditorFrame({
   )
   const editorRef = useRef<CodeEditor>(null)
 
-  function toggleFullscreen() {
-    setFullscreen(!isFullscreen)
-  }
+  const toggleFullscreen = useCallback(() => setFullscreen(!isFullscreen), [
```

Comment:
Went into the code to understand why we need this useCallback. But it seems we no longer need the isFullScreen for the useEffect() any more.

Reply:
To be honest I was just following the hints from this plugin to get some of these working, so do need looking at.

Reply:
so remove `isFullScreen` in this file or all files?

Reply:
see the comment bellow